src/gt4py/next/ffront/ast_passes/fix_missing_locations.py

```diff
@@ -42,15 +42,15 @@ def generic_visit(self, node: ast.AST) -> ast.AST:
             node = copy.copy(node)
             parent_node = self._parent_nodes[-1]
 
-            node.lineno = parent_node.lineno
-            node.col_offset = parent_node.col_offset
+            node.lineno = parent_node.lineno  # type: ignore[attr-defined] # this is monkey patching as not all `ast.AST` have `lineno` etc
```

Comment:
```suggestion
            node.lineno = parent_node.lineno  # type: ignore[attr-defined] # setting the attr even if it is not defined is what we want here
```
Monkey patching is updating the behavior of code at run-time here we are only adding an attribute.